build/cluster.js

```diff
@@ -0,0 +1,218 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Gulp tasks for kubernetes cluster management.
+ */
+import childProcess from 'child_process';
+import del from 'del';
+import gulp from 'gulp';
+import chmod from 'gulp-chmod';
+import gulpFilter from 'gulp-filter';
+import git from 'gulp-git';
+import gunzip from 'gulp-gunzip';
+import untar from 'gulp-untar';
+import gulpUtil from 'gulp-util';
+import pathExists from'path-exists';
+import request from 'request';
+import source from 'vinyl-source-stream';
+
+import conf from './conf';
+
+
+let clusterProcess = null,
+    exec = childProcess.exec,
+    isClusterRunning = false,
```

Comment:
Just one more thing. Some of the variables here need documentation or typing. I.e., "clusterProcess". What is the type of this variable and why it is null and when it is not. Take a look at `serve.js` for this type of comment: `/**
 * Currently running backend process object. Null if the backend is not running.
 *
 * @type {?child.ChildProcess}
 */`

Also, move all constants to `const` block. EcmaScript6 has such feature, so let's use it.